lib/core/src/Cardano/Wallet/Primitive/Types/UTxOIndex/Internal.hs

```diff
@@ -234,7 +234,7 @@ toList = fold (\ubs u b -> (u, b) : ubs) []
 -- Consider using 'fold' if your goal is to consume all entries in the output.
 --
 toUTxO :: UTxOIndex u -> Map u TokenBundle
-toUTxO = utxo
+toUTxO = universe
```

Comment:
`utxos`? `allUTxOs`? `rawUTxO`? Maybe `universe` is a bit too dressed up

Reply:
> `utxos`? `allUTxOs`? `rawUTxO`? Maybe `universe` is a bit too dressed up

I take your point. 😄 

How about `everything`?